docs/Displaying a User Interface/Main Menu/index.md

```diff
@@ -16,6 +16,12 @@ Every template has a main menu button. The position of this button varies betwee
 On some newer head units, you may have the option to display menu items as a grid of tiles instead of the default list layout. To determine if the head unit supports the tiles layout, check the `SystemCapabilityManager`'s !@@![iOS]`defaultMainWindowCapability.menuLayoutsAvailable`!@@![android,javaEE,javaSE] `getDefaultMainWindowCapability().getMenuLayoutsAvailable()`!@@![iOS, android, javaEE, javaSE] property after successfully connecting to the head unit. To set the menu layout using the screen manager, you will need to set the `ScreenManager.menuConfiguration` property.
 !@
 
+#### List Menu Layout
+![Generic - Menu Appearance List Layout](assets/Generic_list_layout_dark.png)
```

Comment:
For this one, I think it should replace the "Menu Template" image and section (the note in that section can go into the introduction, if that doesn't make sense, just message me). It should match the Main Menu UI image we have in that PR. The grid image is fine as is I think.